innopoints/schemas/variety.py

```diff
@@ -32,6 +38,7 @@ def create_stock_change(self, data, **kwargs):  # pylint: disable=unused-argumen
 
     @pre_load
     def wire_color_size(self, data, **kwargs):  # pylint: disable=unused-argument
+        """Rename `size` to `size_id`, `color` to `color_value`, and normalize the color."""
```

Comment:
but why not just change the model itself?

Reply:
This will come later, when we merge this

Reply:
Well, then maybe add a TODO so that it is not forgotten after the model is changed